README.md

```diff
@@ -387,25 +387,36 @@ spec:
     metadata:
       labels:
         apps: YourAppNameHere
-        audit.kubernetes.io/allow-privilege-escalation: "YourReasonForOverrideHere"
+        audit.kubernetes.io/pod/allow-privilege-escalation: "YourReasonForOverrideHere"
+        container.audit.kubernetes.io/<container-name>/allow-privileged: "YourReasonForOverrideHere"
 ```
 
 Any label with a non-nil reason string will prevent `kubeaudit` from throwing the corresponding error and issue a warning instead.
 Reasons matching `"true"` (not case sensitive) will be displayed as `Unspecified`.
-
-`kubeaudit` supports many labels:
-- [audit.kubernetes.io/allow-privilege-escalation](#allowpe_label)
-- [audit.kubernetes.io/allow-privileged](#priv_label)
-- [audit.kubernetes.io/allow-capability](#caps_label)
-- [audit.kubernetes.io/allow-run-as-root](#nonroot_label)
-- [audit.kubernetes.io/allow-automount-service-account-token](#sat_label)
-- [audit.kubernetes.io/allow-read-only-root-filesystem-false](#rootfs_label)
+Override label can be set on each containers or on a pod.
+
+`kubeaudit` supports many labels on pod or container level:
```

Comment:
I think we can make the doc a little less convoluted if we start with a longer explanation of how override labels can both be attached to a specific container or to a pod and then just describe one of them and then people can swap container-name for pod or the other way round.

Reply:
I have added longer explanation on how pod level and container level override label work